myapp/spec/models/task_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  before do
+    @user = create(:user)
+  end
```

Comment:
what is this `@user` for?

Reply:
Task is associate with user, so need create a user before. But no need for define a user variable. I have remove it.

Reply:
in factory bot, you can create a associated record with `before`, `after`... ref: https://stackoverflow.com/a/71482271

Please try to use that and remove unneeded user creation in every block.

Reply:
I have refactor the `user` and  `task` model spec, please have a look.